<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 150, 150);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>50</y>
     <width>581</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29, 30, 30);
border: 2px solid;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>16</width>
     <height>611</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>660</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="Welcome">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 32px;
font-family: Tahoma;
font-weight: 900;
color: white;</string>
   </property>
   <property name="text">
    <string>Welcome back!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Routeselect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: 500;
font-family: Tahoma;
color: white;
font-size: 24px;</string>
   </property>
   <property name="text">
    <string>Which bus route would you be interested in?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="U1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-weight:1000;
font-size: 36px;
background-color: rgb(200, 200, 200);
color: white;</string>
   </property>
   <property name="text">
    <string>U1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="U2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-weight:1000;
font-size: 36px;
background-color: rgb(200, 200, 200);
color: white;
</string>
   </property>
   <property name="text">
    <string>U2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showall">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>113</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-weight:800;
font-size: 22px;
background-color: rgb(200, 200, 200);
color: white;</string>
   </property>
   <property name="text">
    <string>Show All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>641</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: 500;
background-color: rgb(200, 200, 200);
color: white;
</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Details_title">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Tahoma;
font-size: 24px;
color: white;
font-weight: 500;
</string>
   </property>
   <property name="text">
    <string>Get details for individual bus stops</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
color: white;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'tahoma'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:18pt;&quot;&gt;Get additional details for each bus stop, with timetabled bus arrival times, predicted arrival times, and predicted time to reach other bus stops&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_more_info">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
background-color: rgb(200, 200, 200);
font-weight: 300;
color: white;</string>
   </property>
   <property name="text">
    <string>Get more information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
